### PR TITLE
0.3.2 release + docs cleanup

### PR DESCRIPTION
**Pull request by miki725**
- [x] Followed the steps in the contributor's guide: https://crashoverride.com/docs/other/contributing#filing-the-pull-request
- [x] PR title uses [semantic commit messages](https://nitayneeman.com/posts/understanding-semantic-commit-messages-using-git-and-angular/#fix)
- [x] Filled out the template to a useful degree
- [x] https://github.com/crashappsec/nimutils/pull/42
- [x] after nimutils is merged bump con4m here

## Issue

chalk repo has outdated docs which is confusing where docs should be fixed

## Description

* most of the docs were out of date and just added to the confusion where a doc should be updated - in this repo or in the docs site repo
* during release process docs will be copied to chalk so that chalk binary can ship with embedded docs as expected.
* moved release notes to `CHANGELOG.md` in the root of the repo. This is a common pattern in GitHub and will make discovering changelog in the repo a lot easier. To be able to embed it in the chalk binary, `make` ensures to copy it to `src/docs` on build time.

## Testing

no functional change. just build process change

### CODE REVIEW

File: src/commands/cmd_help.nim [hunk 1]
```diff
@@ -520,8 +526,10 @@ proc runChalkHelp*(cmdName = "help") {.noreturn.} =
         toOut += con4mRuntime.getHelpTopics()
       of "builtins":
         toOut = con4mRuntime.getBuiltinsTableDoc()
+      of "changelog", "release", "releases":
+        toOut += getEmbeddedDoc("changelog")
       else:
-        let toCheck = [arg, "core-" & arg, "howto-" & arg, "guide" & arg]
+        let toCheck = [arg, arg & ".md" & "core-" & arg, "howto-" & arg, "guide" & arg]
```
Comment (MyNameIsMeerkat) [inline src/commands/cmd_help.nim:532]: are you missing a comma `,` here? i think this line should be as follows correct? With a `,` after the new item of `arg & ".md"` you are looking to match on ?

```
let toCheck = [arg, arg & ".md", & "core-" & arg, "howto-" & arg, "guide" & arg]
```

**Review by MyNameIsMeerkat — CHANGES_REQUESTED** [review]
Looks good but I think there may be one small typo mistake in your changes to `cmd_help.nim`, think you are missing a comma `,` after the new item you inserted into the list. As currently written two adjacent list items get concatenated

File: src/commands/cmd_help.nim (hunk 1 above)
Comment (miki725) [reply src/commands/cmd_help.nim]: good spot. thanks. updated

**Review by MyNameIsMeerkat — APPROVED** [review]
Commas looking good to me :D

**Review by viega — APPROVED** [review]
👋📜

**Pull request merged**